data/src/main/java/com/linkedin/data/codec/symbol/DefaultSymbolTableProvider.java

```diff
@@ -47,6 +47,11 @@
    */
   private static final String ACCEPT_HEADER = "Accept";
 
+  /**
+   * Symbol table request header
+   */
+  private static final String SYMBOL_TABLE_HEADER = "X-Restli-Symbol-Table-Request";
```

Comment:
It needs to be "X-RestLi", Capitalize L

Reply:
HTTP Headers are case insensitive, and h2 actually sends them as lower case. I would suggest lowercasing everything to save some garbage :D

Reply:
thanks will do